editors/vscode/src/common.ts

```diff
@@ -181,28 +210,28 @@ export class PropertiesViewProvider implements vscode.WebviewViewProvider {
                 command: "show_welcome",
                 message: "The active editor does not contain a Slint file.",
             });
+            refresh(); // Language Information might not be available yet!
             return;
         }
         if (client === null) {
             this._view?.webview.postMessage({
                 command: "show_welcome",
                 message: "Waiting for Slint LSP",
             });
+            refresh();
```

Comment:
We can call `refresh_view` from the code that that sets the client.

Reply:
We already call the refresh function from `onDidChangeActiveTextEditor(` , is that not enough?

and for the client becoming non-null, this could be handled in `set_client` in this file too